e2e/test/scenarios/dashboard/reproductions/31274-dashcard-actions-overlap.cy.spec.js

```diff
@@ -0,0 +1,79 @@
+import { editDashboard, restore, visitDashboard } from "e2e/support/helpers";
+
+const baseTextDashcardDetails = {
+  col: 0,
+  row: 0,
+  size_x: 2,
+  size_y: 2,
+  visualization_settings: {
+    virtual_card: {
+      name: null,
+      display: "text",
+      visualization_settings: {},
+      dataset_query: {},
+      archived: false,
+    },
+    text: "Text card",
+  },
+};
+
+const createTextCards = length => {
+  return Array.from({ length }).map((_, index) => {
+    return {
+      ...baseTextDashcardDetails,
+      id: index + 1,
+      // reversed order of appearing
+      row: (length - index - 1) * 2,
+      visualization_settings: {
+        ...baseTextDashcardDetails.visualization_settings,
+        text: `Text card ${index + 1}`,
+      },
+    };
+  });
+};
+
+describe("issue 31274", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  // cypress automatically scrolls to the element, but we don't need it in this test
+  it(
+    "should not clip dashcard actions (mestabase#31274)",
+    { scrollBehavior: false },
```

Comment:
@nemanjaglumac what do you think about such approach of avoiding `scroll` in the view? I'd like to see all dashcards in the viewport, but if I just use realHover or trigger mouseover, cypress scrolls and hides first dashcard